assets/js/jquery.quicksearch.js

```
@@ -0,0 +1,181 @@
+(function($, window, document, undefined) {
```

Biome (1.9.4)

[error] Do not shadow the global "undefined" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)